.claude/skills/commit-push-pr/SKILL.md

```
@@ -0,0 +1,104 @@
+---
+name: commit-push-pr
+description: 프로젝트 컨벤션에 따라 커밋하고, (요청 시) 푸시 및 PR을 생성할 때 로드. 커밋 메시지 포맷, 브랜치 전략, PR 규칙을 자동 적용.
+---
+
+> **언어**: 이 작업의 사용자 대상 응답은 모두 한국어로 작성한다. (코드, 식별자, 로그 등 기술적 산출물은 제외.)
```

markdownlint-cli2 (0.23.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)